cluster/gce/trusty/node.yaml

```diff
@@ -262,7 +262,7 @@ script
 			echo "Docker daemon failed!"
 			pkill docker
 		fi
-		if ! curl --insecure -m ${max_seconds} -f -s https://127.0.0.1:10250/healthz > /dev/null; then
+		if ! curl --insecure -m ${max_seconds} -f -s https://127.0.0.1:${KUBELET_PORT}/healthz > /dev/null; then
```

Comment:
This doesn't handle proper defaulting if KUBELET_PORT isn't explicitly specified.